src/css/catalog/CatalogCard.css

```
@@ -0,0 +1,106 @@
+#course-list {
+  flex-wrap: wrap;
+  color: var(--offwhite);
+  font-family: var(--fontfamily);
+}
+
+#course-item {
```

As mentioned in the Card component itself, they are repeated and so not unique, so this should be a `class` not an `id`
```suggestion
.course-item {
```